homeassistant/components/vera/manifest.json

```diff
@@ -3,6 +3,6 @@
   "name": "Vera",
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/vera",
-  "requirements": ["pyvera==0.3.11"],
+  "requirements": ["pyvera==0.3.12"],
```

Comment:
Should this be 0.3.13?

Reply:
Good catch - thanks  (when testing I install the library directly - so would have missed this).